src/styles/main.scss

```
@@ -1,3 +1,49 @@
 body {
   margin: 0;
 }
+
+.box {
+  color: white;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  grid-template-rows: 300px;
+  grid-auto-rows: 300px;
+}
+
+@media (min-width: 900px) {
+  .box {
+    grid-template-columns: repeat(3,minmax(300px,1fr));
+  }
+}
+
+.snake {
+  display: grid;
```

```suggestion
  display: flex;
```
флекс будет достаточно, ты не используешь возможности grid